asctb-api/src/functions/api.functions.ts

```diff
@@ -83,9 +90,13 @@ const buildMetadata = (metadataRows: string[][], warnings: Set<string>): Record<
         metadataKey = metadataIdentifier.toLowerCase();
         warnings.add(`WARNING: unmapped metadata found ${metadataIdentifier}`);
       }
-      metadata[metadataKey] = metadataValue.split(DELIMETER).map(item => item.trim()).join(', ');
+      if (metadataValue.includes(DELIMETER)) {
```

Comment:
This won't work if they have only one author or one of anything. Need to just set for each metadata key whether you will interpret it as a single or mulitple value metadata item.

Reply:
created separate array to identify array fields in metadata